ext/pdo_pgsql/tests/issue78621.inc

```diff
@@ -1,20 +1,31 @@
 <?php
 require_once dirname(__FILE__) . '/../../../ext/pdo/tests/pdo_test.inc';
 require_once dirname(__FILE__) . '/config.inc';
-$db = PDOTest::test_factory(dirname(__FILE__) . '/common.phpt');
+if (!isset($db)) {
+$db = new PdoPgsql($config['ENV']['PDOTEST_DSN']);
```

Comment:
```suggestion
    $db = new PdoPgsql($config['ENV']['PDOTEST_DSN']);
```

Reply:
Oops, leftovers of the first unconditional, flat version.